proto/Dockerfile

```diff
@@ -17,8 +20,13 @@ RUN set -eux; \
       golang \
       nodejs \
       npm \
+      unzip \
       ; \
     go install google.golang.org/protobuf/cmd/protoc-gen-go@${PROTOC_GEN_GO_VERSION}; \
-    go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@${PROTOC_GEN_GO_GRPC_VERSION}
+    go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@${PROTOC_GEN_GO_GRPC_VERSION}; \
+    go install github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc@${PROTOC_GEN_DOC_VERSION}; \
```

Comment:
I would suggest downloading from the github releases and doing checksum as opposed to doing a go install version of it in `proto/01-protoc-base/install.sh`; I don't think this Dockerfile needs to be edited?